spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2864Test.java

```diff
@@ -0,0 +1,37 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.BugCollection;
+import edu.umd.cs.findbugs.test.SpotBugsRunner;
+import edu.umd.cs.findbugs.test.SpotBugsExtension;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.nio.file.Paths;
+
+/**
+ * @author gtoison
+ *
+ */
+@ExtendWith(SpotBugsExtension.class)
+class Issue2864Test extends AbstractIntegrationTest {
```

Comment:
You don't need to use both `@ExtendWith(SpotBugsExtension.class)` and extend `AbstractIntegrationTest`. Only one of them is enough. I prefer the `AbstractIntegrationTest`, since that's more simple, and https://github.com/spotbugs/spotbugs/pull/2840 solves the missing classes problem for that.

Reply:
Thanks for reviewing, I've made the change